source/fundamentals/atlas-search.txt

```diff
@@ -0,0 +1,606 @@
+
+.. literalinclude:: /includes/fundamentals/code-examples/atlas-search/AtlasSearchExamples.cs
+   :start-after: // start-regex-search
+   :end-before: // end-regex-search
+   :language: csharp
+   :dedent:
+
+The search returns the following results:
+
+.. code-block:: json
+
+   { "_id" : 1, "make" : "Fender", "description" : "Classic guitars known for their versatility.", "establishedYear" : 1946, "in-stock" : true, "rating" : 9 }
+   { "_id" : 2, "make" : "Gibson", "description" : "Classic guitars known for their rich, full tones.", "establishedYear" : 1902, "in-stock" : true, "rating" : 8 }
+   { "_id" : 4, "make" : "Kiesel", "description" : "Quality guitars made only for custom orders.", "establishedYear" : 2015, "in-stock" : false, "rating" : null }
+   { "_id" : 5, "make" : "Ibanez", "description" : "Well-crafted guitars used by many professional guitarists.", "establishedYear" : 1957, "in-stock" : true, "rating" : 7 }
+
+.. note::
+
+   By default the ``regex`` operator cannot run on an analyzed field. You can
+   allow it to run on an analyzed field by setting ``allowAnalyzedField`` option
+   to true, as follows:
+
+   .. code-block:: csharp
+
+      var result = guitarsCollection.Aggregate()
+       .Search(Builders<Guitar>.Search.Regex(r => r.Make, regex, true))
+       .ToList();
+
+   Setting the ``allowAnalyzedField`` option to true may lead to unexpected
+   search results. To learn more, see :atlas:`regex Behavior
+   </atlas-search/regex/#behavior>`.
+
+To learn more about the ``regex`` operation, see the :atlas:`Atlas
+regex </atlas-search/regex>` guide.
+
+Span
+~~~~
+
+Use the ``Span`` method to search for text search matches within regions of a
+field. You can use this method to find strings which are near each other to
+specified degrees of precision.
+
+.. note::
+
+   The ``span`` operator is more computationally intensive than other operators,
+   because queries must keep track of positional information.
+
+The following example searches the ``guitars`` collection for documents in which
+the value of the ``description`` field contains the strings "guitars" and
+"quality" within one word of each other.
+
+.. literalinclude:: /includes/fundamentals/code-examples/atlas-search/AtlasSearchExamples.cs
+   :start-after: // start-span-search
+   :end-before: // end-span-search
+   :language: csharp
+   :dedent:
+
+The search returns the following document:
+
+.. code-block:: json
+
+   { "_id" : 4, "make" : "Kiesel", "description" : "Quality guitars made only for custom orders.", "establishedYear" : 2015, "in-stock" : false, "rating" : null }
+
+Although the document with ``_id: 3`` contains the strings "guitars" and
+"quality", they are separated by more than one word, so the search omits this
+document from the results.
+
+To learn more about the ``span`` operation, see the :atlas:`Atlas
+span </atlas-search/span>` guide.
+
+Text
+~~~~
+
+Use the ``Text`` method to search a document for a given string or array of
+strings.
```

Comment:
Question

How is this different from `Phrase`? This seems near identical.

Reply:
I've added a little more to the description here. ``Text`` will search each word in the string separately, ``Phrase`` searches them as a singular phrase.